### PR TITLE
feat(example): add ddp and crop zones metadata

### PR DESCRIPTION
**Pull request by CedrickArmel**
Add ddp and crop zones metadata.

### CODE REVIEW

**Review by Gab-pm — APPROVED** [review]

**Comment by ccamel** [comment]
@CedrickArmel The tests failed. Can you take a look and fix it? Thank you.

File: example/ddp/ddp.ttl
Comment (ccamel) [inline example/ddp/ddp.ttl:10]: I think `data space` should be replaced by `zone`.

File: example/ddp/gouvernance.ttl
Comment (ccamel) [inline example/ddp/gouvernance.ttl:209]: Same here. Use the term `Zone` instead.

**Review by ccamel — CHANGES_REQUESTED** [review]

**Review by Gab-pm — COMMENTED** [review]
I will check with clients if the files can be public before to approve

File: example/ddp/ddp.ttl
Comment (CedrickArmel) [reply example/ddp/ddp.ttl:10]: @ccamel i checked the files, read the test code made several amendment commits and still don't see what still causes the test to fail.

Comment (ccamel) [reply example/ddp/ddp.ttl:10]: @CedrickArmel did you rebased against `main` after #152 merge?

**Comment by CedrickArmel** [comment]
@ccamel I have just made just made a `git rebase origin/main` but nothing changes.

**Comment by ccamel** [comment]
> @ccamel I have just made just made a `git rebase origin/main` but nothing changes.

Need to test locally to see what's going wrong, because the GitHub action logs don't provide the information (should be improved).

**Comment by ccamel** [comment]
I have identified the issue. It appears that there are 2 instances of `meta:Paragraph` with the same identifier, which results in 2 `core:hasNumber` properties with the same subject. Fortunately, this is being detected by the tests.

Fix is currently underway.

**Review by ccamel — APPROVED** [review]
LGTM!

**Comment by ccamel** [comment]
Here we are. @CedrickArmel wait for @Gab-pm decision before merging.

**Review by Gab-pm — APPROVED** [review]

**Comment by ccamel** [comment]
All right! Thanks all for your contribution.

**Pull request merged**
